src/NuGet.Core/NuGet.Packaging/Licenses/NuGetLicenseData.cs

```diff
@@ -0,0 +1,486 @@
+            {"GPL-2.0-with-autoconf-exception", new LicenseData(licenseID: "GPL-2.0-with-autoconf-exception", referenceNumber: 358, isOsiApproved: false, isDeprecatedLicenseId: true) },
+            {"GPL-2.0-with-bison-exception", new LicenseData(licenseID: "GPL-2.0-with-bison-exception", referenceNumber: 359, isOsiApproved: false, isDeprecatedLicenseId: true) },
+            {"GPL-2.0-with-classpath-exception", new LicenseData(licenseID: "GPL-2.0-with-classpath-exception", referenceNumber: 360, isOsiApproved: false, isDeprecatedLicenseId: true) },
+            {"GPL-2.0-with-font-exception", new LicenseData(licenseID: "GPL-2.0-with-font-exception", referenceNumber: 361, isOsiApproved: false, isDeprecatedLicenseId: true) },
+            {"GPL-2.0", new LicenseData(licenseID: "GPL-2.0", referenceNumber: 362, isOsiApproved: true, isDeprecatedLicenseId: true) },
+            {"GPL-3.0+", new LicenseData(licenseID: "GPL-3.0+", referenceNumber: 363, isOsiApproved: true, isDeprecatedLicenseId: true) },
+            {"GPL-3.0-with-GCC-exception", new LicenseData(licenseID: "GPL-3.0-with-GCC-exception", referenceNumber: 364, isOsiApproved: true, isDeprecatedLicenseId: true) },
+            {"GPL-3.0-with-autoconf-exception", new LicenseData(licenseID: "GPL-3.0-with-autoconf-exception", referenceNumber: 365, isOsiApproved: false, isDeprecatedLicenseId: true) },
+            {"GPL-3.0", new LicenseData(licenseID: "GPL-3.0", referenceNumber: 366, isOsiApproved: true, isDeprecatedLicenseId: true) },
+            {"LGPL-2.0+", new LicenseData(licenseID: "LGPL-2.0+", referenceNumber: 367, isOsiApproved: true, isDeprecatedLicenseId: true) },
+            {"LGPL-2.0", new LicenseData(licenseID: "LGPL-2.0", referenceNumber: 368, isOsiApproved: true, isDeprecatedLicenseId: true) },
+            {"LGPL-2.1+", new LicenseData(licenseID: "LGPL-2.1+", referenceNumber: 369, isOsiApproved: true, isDeprecatedLicenseId: true) },
+            {"LGPL-2.1", new LicenseData(licenseID: "LGPL-2.1", referenceNumber: 370, isOsiApproved: true, isDeprecatedLicenseId: true) },
+            {"LGPL-3.0+", new LicenseData(licenseID: "LGPL-3.0+", referenceNumber: 371, isOsiApproved: true, isDeprecatedLicenseId: true) },
+            {"LGPL-3.0", new LicenseData(licenseID: "LGPL-3.0", referenceNumber: 372, isOsiApproved: true, isDeprecatedLicenseId: true) },
+            {"Nunit", new LicenseData(licenseID: "Nunit", referenceNumber: 373, isOsiApproved: false, isDeprecatedLicenseId: true) },
+            {"StandardML-NJ", new LicenseData(licenseID: "StandardML-NJ", referenceNumber: 374, isOsiApproved: false, isDeprecatedLicenseId: true) },
+            {"eCos-2.0", new LicenseData(licenseID: "eCos-2.0", referenceNumber: 375, isOsiApproved: false, isDeprecatedLicenseId: true) },
+            {"wxWindows", new LicenseData(licenseID: "wxWindows", referenceNumber: 376, isOsiApproved: false, isDeprecatedLicenseId: true) },
```

Comment:
nit: can these be sorted alphabetically?

Reply:
I prefer to keep the order the same as they are in https://github.com/spdx/license-list-data/blob/master/json/licenses.json. 

They are sorted by reference number.